src/components/MenuSelector/ToggleMenu.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import './ToggleMenu.css'
+
+function ToggleMenu({ tab, setTab }) {
+
+    return (
+    <div className="toggle-container">
+        <button 
+            className="toggle"
+            onClick={() => setTab('breakfast')}  
+            style={
+                { backgroundColor: tab === 'breakfast' ? '#FFF' : '#DED2FF',
+                  fontWeight: tab === 'breakfast' ? 'bold' : 'normal',
+                  fontSize: tab === 'breakfast' ? '26px' : '25px'
+                }
```

Comment:
Sería más legible realizarlo mediante classname

Ejemplo

Mis clases css

```css
. toogle {
  // ... some styles
  background-color: #DED2FF;
  font-weight: normal;
  font-size: 25px;
}

.toggle--breakfast{
  background-color: #FFF;
  font-weight: bold;
  font-size: 26px;
}
```

Y en mi component

```jsx
<button className={`toggle ${tab === 'breakfast' ? 'toggle-breakfast': ''}`} onClick={() => setTab('breakfast')}>
```suggestion
            style={
                { backgroundColor: tab === 'breakfast' ? '#FFF' : '#DED2FF',
                  fontWeight: tab === 'breakfast' ? 'bold' : 'normal',
                  fontSize: tab === 'breakfast' ? '26px' : '25px'
                }
```

Reply:
Lo intentamos hacer así, pero cuando dabamos click en otro lado de la pantalla (por ejemplo en los elementos para agregarlos al menú), el  botón dejaba de estar activo y la clase cambiaba a la primera :(